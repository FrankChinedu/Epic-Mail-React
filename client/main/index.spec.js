import React from 'react';
import { shallow } from 'enzyme';
import Main from './index';

describe('Main', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Main debug />);

    expect(component).toMatchSnapshot();
  });
});
