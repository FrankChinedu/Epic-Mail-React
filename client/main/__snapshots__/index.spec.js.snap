// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly in "debug" mode 1`] = `
<Fragment>
  <BrowserRouter>
    <Connect(AuthWrapper) />
  </BrowserRouter>
</Fragment>
`;
